libs/core/src/lib/resizable-card-layout/resizable-card-item/resizable-card-item.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="fd-resizable-card-layout__item" [class.fd-resizable-card-layout__indication-border]="showBorder"
+     #resizeCard>
+    <div class="fd-resizable-card-border-display" [style.width.px]="cardWidth" [style.height.px]="cardHeight" (mouseenter)="showResizeIcon($event)"
+        (mouseleave)="hideResizeIcon($event)">
+        <div class="fd-resizable-card-layout__card">
+            <ng-content></ng-content>
+
+            <ng-container *ngIf="showingResizeIcon">
+                <div class="fd-resizable-card-layout__icon-wrapper">
+                    <div class="fd-resizable-card-layout__icon" #cornerHandle (mousedown)="onMouseDown($event, 'both')">
+                    </div>
+                </div>
+            </ng-container>
+        </div>
+
+        <ng-container *ngIf="resizable">
+            <ng-container *ngTemplateOutlet="verticalHandleTpt"></ng-container>
+            <ng-container *ngTemplateOutlet="horizontalHandleTpt"></ng-container>
+        </ng-container>
+    </div>
+    
+    <div class="fd-resizable-card-expansion" [style.width.px]="resizeIndicationBorderWidth" [style.height.px]="cardHeight"></div>
+</div>
+
+<ng-template #verticalHandleTpt>
```

Comment:
Does `Tpt` stand for `template`? Please rename with more meaningful name.

Reply:
corrected.